extensions/blocks/podcast-player/style.scss

```diff
@@ -300,91 +274,65 @@ $player-background: transparent;
 	 */
 	.jetpack-podcast-player__track-error {
 		display: block;
-		margin-left: ($player-grid-spacing - 2px) + $track-status-icon-size + $track-v-padding; // has to be aligned with the track title
-		margin-bottom: $track-h-padding;
-		color: $alert-red;
+		margin-left: ( $gutter-l - 2px ) + $track-status-icon-size + $gutter-m; // has to be aligned with the track title
+		margin-bottom: $gutter-s;
 		font-size: 0.8em;
 		font-weight: normal;
-
-		& > span {
-			color: $text-color;
-		}
-
-		& > span > a {
-			color: inherit;
-		}
-	}
-
-	.has-primary .jetpack-podcast-player__track-error {
-		color: currentColor;
+		color: $jetpack-podcast-player-error;
 
 		& > span {
 			color: var( --jetpack-podcast-player-secondary );
-		}
-	}
 
-	.jetpack-podcast-player__track-status-icon--error {
-		fill: $text-color-error;
+			&:not( .has-secondary ) {
+				color: $jetpack-podcast-player-secondary;
+			}
+		}
 	}
 
-	.has-primary .jetpack-podcast-player__track-status-icon--error {
-		fill: currentColor;
+	// Use primary color to prevent visual conflicts (i.e. red background)
+	.jetpack-podcast-player__track.has-primary .jetpack-podcast-player__track-error {
+		color: inherit;
 	}
 
 	/**
 	 * Global error element, replaces the whole block with the error message.
 	 */
 	.jetpack-podcast-player__error {
-		padding: $player-grid-spacing;
+		padding: $gutter-l;
 		margin: 0;
-		color: $alert-red;
+		color: $jetpack-podcast-player-error;
 		font-size: 0.8em;
 		font-weight: normal;
 	}
 
-	/**
-	 * Style the block to hide dynamic UI and show just its default style.
-	 */
-	&.is-default {
-		.jetpack-podcast-player__track-title {
-			// Change padding to account for missing space for status-icon.
-			padding-left: $player-grid-spacing - $track-v-padding;
-		}
-
-		.jetpack-podcast-player__audio-player {
-			display: none;
-		}
-
-		&.is-default .jetpack-podcast-player__track-status-icon {
-			display: none;
-		}
-	}
-
 	/**
 	 * Style player by overriding mejs default styles
 	 */
 	.mejs-container,
+	.mejs-container .mejs-controls,
 	.mejs-embed,
 	.mejs-embed body,
-	.mejs-container .mejs-controls,
 	.mejs-mediaelement {
-		background-color: $player-background;
+		background-color: transparent;
```

Comment:
I wonder if we should set this as `transparent` or use a `$var` of some kind like before 🤔

Reply:
> I wonder if we should set this as transparent

What do you mean?

Reply:
Like, right now we are using the color `transparent`. Should we set it back to:
`$player-background: transparent;`

`background-color: $player-background;`?

Reply:
I removed the var for 2 reasons:
1. It’s not something that we want to keep as a var at the top so we can easily customize it - it should always be transparent,
2. It’s not used anywhere else.

As a comparison, i.e. the `$current-track-title-font-size` is also used in a single place, but IMO it’s worth having it at the top so we can easily adjust it if needed.